File: maioridade.c
#include <stdio.h>

int main() {
    int idade;
    printf("Digite sua idade: ");
    scanf("%d", &idade);
    if (idade >= 18) {
        printf("Voce eh maior de idade.\n");
    } else {
        printf("Voce nao eh maior de idade.\n");
    }
    return 0;
}

File: maioridade.java
import java.util.Scanner;

public class maioridade {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite sua idade: ");
        int idade = scanner.nextInt();
        if (idade >= 18) {
            System.out.println("Você é maior de idade.");
        } else {
            System.out.println("Você não é maior de idade.");
        }
        scanner.close();
    }
}
